UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetGenerateFilesForProductIdentifierWithoutParameterexchangeSetStandard.cs

```diff
@@ -9,25 +9,25 @@
 namespace UKHO.ExchangeSetService.API.FunctionalTests.FunctionalTests
 {
     [TestFixture]
-    public class ExchangeSetGenerateFilesForProductIdentifierWithIncorrectValueForIsUnencryptedParameter : ObjectStorage
+    public class ExchangeSetGenerateFilesForProductIdentifierWithoutParameterexchangeSetStandard : ObjectStorage
```

Comment:
Please use Pascal Case e.g. exchangeSetStandard => ExchangeSetStandard wherever applicable. Please check the below change as below at other places as well.
```suggestion
    public class ExchangeSetGenerateFilesForProductIdentifiersWithoutExchangeSetStandardParameter : ObjectStorage
```

Reply:
Resolved.Thanks